content/docs/tech-insights/introduction/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Introduction
+publishedDate: '2022-11-15'
+description: An introduction about Tech Insights plugin.
+---
+
+## Introduction
+
+Roadie Tech Insights helps you keep track of all of your software assets and make sure they meet your quality and compliance targets.
```

Comment:
Do we have to always call it "Roadie" Tech Insights? Seems a bit redundant. Are you trying to disambiguate from OSS Tech Insights?

Reply:
Yeah, I think we need to somehow separate the two

Reply:
It's fine for now but this is making me think we need to call it Tech Insights Pro.

/cc @jorgelainfiesta